sdcm/cluster.py

```diff
@@ -3361,6 +3369,14 @@ def get_all_tables_with_cdc(self, db_node: BaseNode) -> List[str]:
 
         return ks_tables_with_cdc
 
+    def run_node_benchmarks(self):
+        if not self.params.get("run_db_node_benchmarks") or not self.nodes:
+            return
+
+        self.node_benchmark_manager.add_nodes(self.nodes)
+        self.node_benchmark_manager.install_benchmark_tools()
+        self.node_benchmark_manager.run_benchmarks()
```

Comment:
I think that one of the main things here is missing.
The comparison between the nodes and sending a warning event (for now).
I don't know what are the most relevant metrics we would like to follow, but we need one or more metrics that we care about and compare.

Reply:
Agreed that a comparison and warning can be useful. That being said I'd rather do the node fairness comparison and a run fairness comparison (node metrics for the same type of instance across runs) in a separate PR. It would be easier to review on it's own.

Reply:
We can postpone this one to the next PR, just open a task for it.